specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-11-01/examples/ViewRender.json

```diff
@@ -0,0 +1,75 @@
+{
+  "parameters": {
+    "api-version": "2019-11-01",
+    "scope": "subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MYDEVTESTRG",
+    "parameters": {
+      "eTag": "\"1d4ff9fe66f1d10\"",
+      "properties": {
+        "displayName": "swagger Example",
+        "query": {
+          "type": "Usage",
+          "timeframe": "MonthToDate",
+          "dataset": {
+            "granularity": "Daily",
+            "aggregation": {
+              "totalCost": {
+                "name": "PreTaxCost",
+                "function": "Sum"
+              }
+            },
+            "grouping": [],
+            "sorting": [
+              {
+                "direction": "Ascending",
+                "name": "UsageDate"
+              }
+            ]
+          }
+        },
+        "chart": "Table",
+        "accumulated": "true",
+        "metric": "ActualCost",
+        "kpis": [
+          {
+            "type": "Forecast",
+            "id": null,
+            "enabled": true
+          },
+          {
+            "type": "Budget",
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MYDEVTESTRG/providers/Microsoft.Consumption/budgets/swaggerDemo",
+            "enabled": true
+          }
+        ],
+        "pivots": [
+          {
+            "type": "Dimension",
+            "name": "ServiceName"
+          },
+          {
+            "type": "Dimension",
+            "name": "MeterCategory"
+          },
+          {
+            "type": "TagKey",
+            "name": "swaggerTagKey"
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MYDEVTESTRG/providers/Microsoft.CostManagement/views/swaggerExample",
+        "name": "swaggerExample",
+        "type": "donwloadlink",
+        "eTag": "\"1d4ffa5a9c2430c\"",
+        "properties": {
+          "imageUrl": "https://storagepilot.blob.core.windows.net/dashboardimage/2020/10/19/5/31/eff0d624-8630-4ced-bd9b-971df5878fcb?sv=2016-05-31",
+          "errorMessage": ""
```

Comment:
In what scenario would an errorMessage be returned with a 200 response?

Reply:
Very valid point. Change reverted.